component/src/main/java/me/androidbox/component/event/VisitorSection.kt

```diff
@@ -15,34 +17,94 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import me.androidbox.component.R
+import me.androidbox.component.agenda.VisitorFilterType
 import me.androidbox.component.ui.theme.BusbyTaskyTheme
 import me.androidbox.component.ui.theme.visitorTextFontColor
+import java.util.UUID
+import kotlin.random.Random
 
 @Composable
 fun VisitorSection(
+    visitors: List<VisitorInfo>,
     modifier: Modifier = Modifier,
-    @StringRes titleResId: Int,
-    content: @Composable () -> Unit
+    visitorDeleteClicked: (VisitorInfo) -> Unit,
+    visitorFilterType: VisitorFilterType = VisitorFilterType.ALL,
 ) {
-    Column(modifier = modifier
-        .fillMaxWidth()) {
+    Column(modifier = modifier) {
+        Spacer(modifier = Modifier.height(16.dp))
 
-        Text(
-            text = stringResource(id = titleResId),
-            fontWeight = FontWeight.Medium,
-            fontSize = 16.sp,
-            color = MaterialTheme.colorScheme.visitorTextFontColor)
+        LazyColumn(modifier = Modifier
+            .fillMaxWidth()
+            .height(300.dp),
```

Comment:
@MKiperszmid 
I was having issues. I think because the outer column can't know the exact height because of the nested lazycolumn. Without this no visitors are shown.

Reply:
Yes, I'd put the whole event screen in a single LazyColumn

Reply:
@philipplackner 
Just wondering how that would work. My EventScreen has something like this below. I am wondering which column would be the best candidate for a LazyColumn. And just wondering what advantages of having a LazyColumn over a conventional Column? 
```
 Scaffold(
        modifier = modifier,
        topBar = {
            AgendaDetailTopBar()
        },
        content = {
            Column() {

                Column () {
                      AgendaHeader()

                      PhotoPicker()

                      AgendaDuration()

                      AgendaDropDownMenu()

                      AlarmReminder()
                    
                      VisitorFilter()

                      VisitorSection()

                      AgendaAction()
                }
       }
}
```

Reply:
Yes correct, you'd use individual item blocks. Just for the attendees list make sure to wrap these in `items` instead, otherwise the whole list will be treated as one composable and you won't benefit from the lazy loading behavior.